code/modules/keybindings/keybind/living.dm

```diff
@@ -11,6 +11,12 @@
 	full_name = "Resist"
 	description = "Break free of your current state. Handcuffed? on fire? Resist!"
 
+/datum/keybinding/living/rest
+	hotkey_keys = list("V")
+	name = "rest"
+	full_name = "Rest"
+	description = "Toggle between lying down or standing up."
+
```

Comment:
need /down() behavior

Reply:
Make sure I called the right proc on this one.